docs/bin/build-and-mirror.sh

```diff
@@ -2,17 +2,44 @@
 #
 # Builds the HTML documentation tree and mirrors it to the geomesa.github.io repository.
 
-# IMPORTANT: Update these paths for your system!
+# default paths for docs, can be overridden by command line switches
 GEOMESA=/opt/devel/src/geomesa
```

Comment:
Also, these should respect existing ENV entries like this:
```GEOMESA=/path/to GEOMESA_GITHUB_IO=/path/to/this build_and_mirror.sh``` so that devs can set it on the command line or in their bashrc env

I actually prefer not to have /opt/devel in here at all :angel: